run.sh

```diff
@@ -44,7 +44,7 @@ if [ "${SMTP_PORT}" = "465" ]; then
 fi
 
 # Create sasl_passwd file with auth credentials
-if [ ! -f /etc/postfix/sasl_passwd ]; then
+if [ ! -f /etc/postfix/sasl_passwd -a ! -z "${SMTP_USERNAME}" ]; then
```

Comment:
If SMTP_USERNAME is set then SMTP_PASSWORD needs to be set too if not you will leave a broken sasl_passwd file. If you check for it on line 21 then when execution reaches here SMTP_PASSWORD will be set so no issues.